src/components/Navbar/Login/Login.component.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useContext } from 'react';
+import { message, Modal, Row } from 'antd';
+import { UserInput, PasswordInput, ButtonStyle } from './Login.styled';
+import { StateContext } from '../../../utils/State';
+import { ConfigContext } from '../../../utils/ConfigState';
+
+const SesionInfo = {
+    User: "User",
+    Password: "123"
+};
+
+const onTextChange = (callback) => (event) => {
+    const { value } = event.target;
+    callback(value);
+};
+
+const Login = ({ onClose, visible }) => {
+
+    const { state: { Theme: { PrimaryColor, TextColor }, DarkMode } } = useContext(ConfigContext)
+    const { dispatchS } = useContext(StateContext);
+
+    const [User, setUser] = useState("");
+    const [Password, setPassword] = useState("");
```

Comment:
I recommend having Uppercase for components only in React context, it is good if you want to denote something, but, in React's context is used mostly for components